### PR TITLE
#158755847  Embed social media content on footer of landing page

### PR DESCRIPTION
**Pull request by ddouglasz**
### What does this PR do?
- Embed social media content to the application.

### Description of Task to be completed?
- Show trends on discussions about the asgard reaction commerce website on sicial media and its latest tweets directly from the twitter site.

### How should this be manually tested?
- https://asgard-rc-staging.herokuapp.com/

### Any background context you want to provide?
- No

### What are the relevant pivotal tracker stories?
- #158755847 

### Screenshots (if appropriate)
image

### Questions:
- No

### CODE REVIEW

**Comment by Daramola98** [comment]
@ddouglasz please use the PR naming convention that can be found in the project wiki

**Comment by abdulfataiaka** [comment]
@ddouglasz, please remove all commented out codes

File: imports/plugins/custom/landingPage/client/components/MyStoreFront.js [hunk 1]
```diff
@@ -4,6 +4,24 @@ import "../styles/style.css";
 
 
 class MyStoreFront extends getRawComponent("Products") {
+  componentDidMount() {
+    const scriptNode = document.getElementById("twitter-wjs");
+    if (scriptNode) {
+      scriptNode.parentNode.removeChild(scriptNode);
+    }
+
+    !(function (d, s, id) {
```
Comment (abdulfataiaka) [inline imports/plugins/custom/landingPage/client/components/MyStoreFront.js]: @ddouglasz, please use descriptive variable names

File: imports/plugins/custom/landingPage/client/components/MyStoreFront.js [hunk 2]
```diff
@@ -128,6 +146,14 @@ class MyStoreFront extends getRawComponent("Products") {
             <div className="row">
               <div className="col-md-3">
                 <h2 className="lead">ASGARD MART</h2>
+
+
+                <div className="col-lg-4 col-md-4 col-sm-6 col-xs-12 feed-container">
+
+                  {/* <a className="twitter-timeline" href="https://twitter.com/dooghieshrek?ref_src=twsrc%5Etfw">Tweets by dooghieshrek</a> */}
+
```
Comment (abdulfataiaka) [inline imports/plugins/custom/landingPage/client/components/MyStoreFront.js]: Remove commented codes

File: imports/plugins/custom/landingPage/client/templates/MyStoreFront.js [hunk 3]
```diff
@@ -8,3 +8,20 @@ Template.landingPage.helpers({
     };
   }
 });
+
```
Comment (abdulfataiaka) [inline imports/plugins/custom/landingPage/client/templates/MyStoreFront.js]: Remove all commented codes

File: imports/plugins/custom/landingPage/client/templates/footer.html [hunk 4]
```diff
@@ -32,4 +48,25 @@ <h5 className="lead">Contact Us</h5>
       </div>
     </div>
   </div>
-</template>
\ No newline at end of file
+
+  <!-- <script>window.twttr = (function(d, s, id) {
```
Comment (abdulfataiaka) [inline imports/plugins/custom/landingPage/client/templates/footer.html]: Remove all commented codes

File: imports/plugins/custom/landingPage/client/templates/footer.html [hunk 5]
```diff
@@ -32,4 +48,25 @@ <h5 className="lead">Contact Us</h5>
...
+      t._e.push(f);
+    };
+  
+    return t;
+  }(document, "script", "twitter-wjs"));</script> -->
+</template>
+<!-- <script dangerouslySetInnerHTML={{ __html: `window.twttr=( function(d, s, id) { var js, fjs=d .getElementsByTagName(s)[0],
+  t=w indow.twttr || {}; if (d.getElementById(id)) return t; js=d .createElement(s); js.id=i d; js.src="https://platform.twitter.com/widgets.js"
+  ; fjs.parentNode.insertBefore(js, fjs); t._e=[ ]; t.ready=f unction(f) { t._e.push(f); }; return t; }(document, "script",
+  "twitter-wjs"));`, }} /> -->
```
Comment (abdulfataiaka) [inline imports/plugins/custom/landingPage/client/templates/footer.html]: Add a new line after line `72`

**Comment by abdulfataiaka** [comment]
@ddouglasz, Please ensure to follow convections with your commit message and PR title and description

File: imports/plugins/custom/landingPage/client/templates/footer.html (hunk 5 above)
Comment (clintfidel) [reply imports/plugins/custom/landingPage/client/templates/footer.html]: remove commented codes @ddouglasz

Comment (ddouglasz) [reply imports/plugins/custom/landingPage/client/templates/footer.html]: On it guys. Thanks for the extra eyes 👀👀

**Comment by clintfidel** [comment]
@ddouglasz Awesome job. Please address all comments on PR.

**Comment by OlatunjiYso** [comment]
Impressive job @ddouglasz , do you have any reasons for making the "ASGART MART" header blue
as against the grey headers on the footer?

**Comment by clintfidel** [comment]
@ddouglasz , for a better user experience it would make sense if the social embedding is done on the right-hand side of the footer as opposed to its current position, the reason for this is users actively react to activities on the right-hand side of pages more often

**Comment by ddouglasz** [comment]
@clintfidel very good observation. Kindly noted and implemented. Thank you for that.

**Comment by ddouglasz** [comment]
@Olatunji, If you check the caption in the header image, You would see where `Always in Style ` was written in blue... The `ASGARD MART` color is to maintain that consistency in the footer too. Nice observation though.

**Pull request merged**